packages/run-protocol/test/vaultFactory/test-liquidator.js

```diff
@@ -106,6 +106,7 @@ test.before(async t => {
       chargingPeriod: 2n,
       recordingPeriod: 6n,
     },
+    minInitialLoan: 5_000_000n,
```

Comment:
Please use an actual Amount here (like with the initial liquidity below). That was clarifying when I redid the test setup for `test-liquidator`

Reply:
I used to have an actual Amount. Check the final commit to see.   That didn't work well with econ-behaviors where I needed a default value for some tests in `.../packages/solo/` that didn't set anything.  The default there can't be an amount because the brand isn't available during initialization.